controllers/base_controller.py

```diff
@@ -260,3 +260,13 @@ def _require_team_admin_access(self, team_number, redirect_url=None):
             TeamAdminAccess.expiration > now).fetch()
         if not existing_access:
             return self.abort(403)
+
+    def _require_user(self):
```

Comment:
We do this check in `UserNotificationBroadcast` as well, but that's the only other place. I'll probably migrate that controller as well. It's also pretty poorly named since it only manages pings.

Reply:
It's not 100% obvious from the name what this function is intended to do.